### PR TITLE
Feat: Add grid component

### PR DESCRIPTION
**Pull request by starver20**
Added grid component, can be used to create
1) Fixed 2 rows or columns
2) Fixed 3 rows or columns
3) Sample page layout
4) Free-flow (as many number as required) rows or columns

[CSB](https://7bgwn.csb.app/components/simplifiedGrid/simplifiedGrid.html) for reference

### CODE REVIEW

**Review by irfan-nawaz — APPROVED** [review]
looks clean nice work 👍🏼

File: components/simplifiedGrid/simplifiedGrid.html [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+          </ul>
+        </aside>
+        <div class="main-content-container">
+          <div class="main-content">
+            <h1 class="component-title">Grid</h1>
+            <p class="component-info">
+              A modal is a window laid over the main window to attract full
+              attention of the user. It can be dismissed by either clicking on
+              the buttons or the backdrop
+            </p>
```
Comment (shaik-saad) [inline components/simplifiedGrid/simplifiedGrid.html]: Here it has a modal description, I guess you've not changed the description here.

**Review by shaik-saad — CHANGES_REQUESTED** [review]
I have added a review, do look into it.

File: components/simplifiedGrid/simplifiedGrid.html (hunk 1 above)
Comment (starver20) [reply components/simplifiedGrid/simplifiedGrid.html]: Thank you, I have changed it.

**Review by shaik-saad — APPROVED** [review]
Looks good, you can go ahead with the merge. Also try to add comments in the html file so that the code can easily be distinguished.

**Comment by starver20** [comment]
> Looks good, you can go ahead with the merge. Also try to add comments in the html file so that the code can easily be distinguished.

Noted, I will follow it, thank you,

**Pull request merged**